components/WinsResources.js

```diff
@@ -42,4 +43,8 @@ console.log(category)
             <br/>
         </div>
     )
-}
\ No newline at end of file
+}
+
+// • Similar to the news page, the hover action triggers the card to expand, with a call to action (Learn more in Line 27) to prompt the user to read/visit the ID page.
+
+// • Ternaries for location and field attributes are commented out for consistent card display in Wins page.
```

Comment:
I think we can just take out these ternaries completely and add them back in the future if we need!